.fixtures.yml

```diff
@@ -0,0 +1,7 @@
+fixtures:
+  repositories:
+    stdlib:
+      repo: 'git://github.com/puppetlabs/puppetlabs-stdlib.git'
+      ref: '4.9.0'
```

Comment:
do we actually need 4.9.0 or will 4.6.0 work? no need to change, i'm just curious